src/settings/_colours-palette.scss

```diff
@@ -34,4 +48,52 @@ $govuk-colours: (
   "grey-3": #dee0e2,
   "grey-4": #f8f8f8,
   "white": #ffffff
+);
+
+/// Modern colour palette
+///
+/// This exists only because you cannot easily set a !default variable
+/// conditionally (thanks to the way scope works in Sass) so we set
+/// `$govuk-colour-palette` using the `if` function.
+///
+/// @access private
+
+$_govuk-colour-palette-modern: (
+  "red": #d4351c,
+  "yellow": #ffdd00,
+  "green": #00703c,
+  "blue": #1d70b8,
+  "dark-blue": #003078,
+  "light-blue": #5694ca,
+  "purple": #4c2c92,
+
+  "black": #0b0c0c,
+  "dark-grey": #6f777b,
+  "mid-grey": #b1b4b6,
+  "light-grey": #f3f2f1,
+  "white": #ffffff,
+
+  "light-purple": #6f72af,
+  "bright-purple": #912b88,
+  "pink": #d53880,
+  "light-pink": #f499be,
+  "orange": #f47738,
+  "brown": #b58840,
+  "light-green": #85994b,
+  "turquoise": #28a197
```

Comment:
I thought Turquoise was being dropped in the new palette. https://github.com/alphagov/govuk-frontend/pull/1245/files#diff-76794ff1589bb2e535aa6b43c4902fb9L27

Reply:
After discussion with @dashouse, we're going to keep most of the 'secondary' colours in the palette as they're useful for teams doing e.g. data visualisation that need a broader palette to work with.